graphblas/io.py

```diff
@@ -550,29 +550,43 @@ def to_pydata_sparse(A, format="coo"):
     if format not in {"coo", "dok", "gcxs"}:
         raise ValueError(f"Invalid format: {format}")
 
-    if format == "gcxs":
-        B = to_scipy_sparse(A, format="csr")
+    if _output_type(A) is _Vector:
+        indices, values = A.to_coo(sort=False)
+        s = COO(indices, values, shape=A.shape)
```

Comment:
btw @SultanOrazbayev I just special-cased `Vector` in `to_pydata_sparse`, b/c previously it was creating a `(1, n)`-shaped sparse array from a Vector instead of a `(n,)`-shaped array.

Reply:
Oops, sorry about that!